workflows/geometry_workflows.md

```diff
@@ -40,11 +40,19 @@ argument. The double loaded corridor leads to space division and thermal zoning
 sliced core and perimeter zoning. It creates a circulation space type running down the center of the
 bar, surrounded by spaces of another space type. For an example, for a primary school the corridor will be paired with the classrooms.
 
-This workflow does not take the actual building footprint into account and does not consider
-the impact of self and context shading.
+![create_bar measure workflow diagram a](../doc_files/create_bar.png)
 
-![urbanopt measure workflow diagram](../doc_files/create_bar.png)
+As shown below workflow does take the actual building footprint into account by creating a second bar when necessary too match the perimeter by orientation non-rectangular geoJSON footprints. Additionally, custom high spaces like school cafeterias are pulled into their own bar from the rest of the building types. This more accurately represents the volume of those spaces. This workflow does not take into account the impact of self and context shading.
 
+![create_bar measure workflow diagram b](../doc_files/create_bar_b.png)
+
+Three additional reporting measures are enabled in this workflow. These reporting measures can be used on any workflow however the QAQC report isn't as meaningful on workflows like the urban geometry which create blended space types instead of recognized standard space types. Below is a brief summary of each report. They all make an HTML file, and they all add some output variable requests into the EnergyPlus IDF file that increases the EnergyPlus SQL file for each building feature.
```

Comment:
```suggestion
Three additional reporting measures are enabled in this workflow. These reporting measures can be used on any workflow; however, the QAQC report isn't as meaningful on workflows like the urban geometry which create blended space types instead of recognized standard space types. Below is a brief summary of each report. They all make an HTML file, and they all add some output variable requests into the EnergyPlus IDF file that increases the EnergyPlus SQL file size for each building feature.
```